File: pr1-cs/Domain/Ticket.cs
namespace pr1_cs.Domain
{
    public class Ticket : Entity<int>
    {
        public Ticket(Game forGame, string clientName)
        {
            ForGame = forGame;
            ClientName = clientName;
        }

        public Ticket(int id, Game forGame, string clientName) : base(id)
        {
            ForGame = forGame;
            ClientName = clientName;
        }

        public Game ForGame { get; set; }

        public string ClientName { get; set; }

        public override string ToString()
        {
            return "Ticket{" +
                   "id='" + Id + '\'' +
                   ", forGame='" + ForGame.Name + '\'' +
                   ", clientName='" + ClientName + '\'' +
                   '}';
        }
    }
}

File: pr1-java/model/src/main/java/pr1Java/model/Ticket.java
package pr1Java.model;

public class Ticket extends Entity<Integer> {
    private Game forGame;
    private String clientName;

    public Ticket() {

    }

    public Ticket(Game forGame, String clientName) {
        this.forGame = forGame;
        this.clientName = clientName;
    }

    public Ticket(Integer id, Game forGame, String clientName) {
        super(id);
        this.forGame = forGame;
        this.clientName = clientName;
    }

    public Game getForGame() {
        return forGame;
    }

    public void setForGame(Game forGame) {
        this.forGame = forGame;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String cost) {
        this.clientName = cost;
    }

    @Override
    public String toString() {
        return "Ticket{" +
                "id='" + getId() + '\'' +
                ", forGame=" + forGame.getName() +
                ", clientName=" + clientName +
                '}';
    }
}
